src/lib/store/store.types.ts

```diff
@@ -241,4 +241,13 @@ export type StoreManager = {
    * @returns The store manager, to allow chaining.
    */
   parseSingleFileIncludes(fileContent: any): StoreManager;
+
+  /**
+   * Gets data from store or from fileSystem if don't exist
+   *
+   * @param relativeFilepath - Relative file path, inside dataDir, to the file to be added.
```

Comment:
* @param relativeFilepath - Relative file path, inside dataDir, to the file to be retrieved.